### PR TITLE
JN: Style Sizing edits and new faqs

### PR DESCRIPTION
**Pull request by jamesnottidge**
I made a few changes:
* Edited copy per marketing request
* Resized by switching to tailwind container
* changed the testimonial assets
* minor style changes across site to improve experience
* Changed the font

### CODE REVIEW

File: app/layout.tsx [hunk 1]
```diff
@@ -11,6 +12,46 @@ import Footer from "@/components/layout/Footer";
...
+      style: "normal",
+    },
+    {
+      path: "../public/proxima-nova-2/Proxima-Nova-Extrabold.otf",
+      weight: "800",
+      style: "normal",
+    }
+  ],
+});
+
```
Comment (bondz) [inline app/layout.tsx:54]: Decagon already has an established typeface that is used on all of its properties, why aren't we using that here?

**Review by bondz — CHANGES_REQUESTED** [review]

**Comment by bondz** [comment]
Also, the testimonial headshots look squished. We don't need to add headshots, especially when they're 30kb each.
Also, need to update the favicon of the site.

**Review by bondz — COMMENTED** [review]
recording

See recording. The text is also too small in some cases. This can be addressed in a separate PR though.

**Comment by jamesnottidge** [comment]
> Also, the testimonial headshots look squished. We don't need to add headshots, especially when they're 30kb each.
> Also, need to update the favicon of the site.

I have updated the testimonial headshots, marketing insisted on them. But we're using animated headshots now
Would update the favicon now

File: app/layout.tsx (hunk 1 above)
Comment (jamesnottidge) [reply app/layout.tsx:54]: The designer didn't use it. I'm just following the design.

**Comment by jamesnottidge** [comment]
I have added OG Metadata, updated the favicon and increased font size

**Pull request merged**